src/status_im/chat/models/input.cljs

```diff
@@ -103,13 +103,17 @@
   Input:  ['/send' 'Jarrad' '1.0']
   Output: '/send Jarrad 1.0'
 
+  Input:  ['/send' '\"Jarrad\"' '1.0']
+  Output: '/send Jarrad 1.0'
```

Comment:
Yes, I actually wanted to create tests for every function in `chat/models/input.cljs`. Will probably do it during the weekend or even earlier.